idunn/api/geocoder.py

```diff
@@ -49,7 +51,11 @@ async def get_intentions():
         if query.lon and query.lat:
             focus = Point(query.lon, query.lat)
 
-        return await nlu_client.get_intentions(text=query.q, lang=query.lang, focus=focus)
+        try:
+            return await nlu_client.get_intentions(text=query.q, lang=query.lang, focus=focus)
+        except NluClientException as exp:
+            logger.warning("Ignored NLU for '%s': %s", query.q, exp.reason, extra=exp.extra)
+            return []
```

Comment:
The new log message does not cover the case when the intention is excluded by `post_filter_intention`. It's probably worth logging them too, as they are intentions we considered irrelevant based on related geocoding results.